From ce14dd9f8cea471c52189586768581454d7f5b21 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:06:13 -0400
Subject: [powerpc] remove unnecessary line continuations

Message-id: <1410545655-205645-145-git-send-email-dzickus@redhat.com>
Patchwork-id: 94157
O-Subject: [RHEL7 PATCH 144/626] powerpc: remove unnecessary line continuations
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 6b3e3b310a884958df030760e725d32a79c0c8b4
Author: Bharat Bhushan <r65777@freescale.com>
Date:   Wed Jun 26 11:12:21 2013 +0530

    powerpc: remove unnecessary line continuations

    Signed-off-by: Bharat Bhushan <bharat.bhushan@freescale.com>
    Signed-off-by: Alexander Graf <agraf@suse.de>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/kernel/process.c b/arch/powerpc/kernel/process.c
index 39f9033..4ce800a 100644
--- a/arch/powerpc/kernel/process.c
+++ b/arch/powerpc/kernel/process.c
@@ -369,7 +369,7 @@ static void set_debug_reg_defaults(struct thread_struct *thread)
  /*
   * Force User/Supervisor bits to b11 (user-only MSR[PR]=1)
   */
- thread->dbcr1 = DBCR1_IAC1US | DBCR1_IAC2US | \
+ thread->dbcr1 = DBCR1_IAC1US | DBCR1_IAC2US |
    DBCR1_IAC3US | DBCR1_IAC4US;
  /*
   * Force Data Address Compare User/Supervisor bits to be User-only
-- 
1.7.1